raiden/tests/utils/factories.py

```diff
@@ -158,7 +158,7 @@ def make_privatekey(privatekey_bin: bytes = EMPTY) -> bytes:
     return if_empty(privatekey_bin, make_privatekey_bin())
 
 
-def make_privatekey_address(
+def make_privkey_address(
```

Comment:
I don't get why this function needs an abbreviation?

Reply:
The new name used to be an alias. Somewhere below, we assigned

`make_privkey_address = make_privatekey_address`

and I wanted to remove one of the two names. Since `make_privatekey_address` is never used I went with `make_privkey_address`.